burrito/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BUILDER_IMAGE=golang:1.16-alpine
```

Comment:
Should we use golang:1.16 without alpine? i.e. this image: https://hub.docker.com/layers/golang/library/golang/1.16/images/sha256-f1cec27767a33f48aaf376d69815f52c9b8202c2d1fd24a7a7778b68f43d1b2f?context=explore for consistency